alacritty/src/display.rs

```diff
@@ -329,6 +340,18 @@ impl Display {
         });
     }
 
+    /// Enable subpixel anti-aliasing
+    fn enable_font_smoothing(enable: bool) {
```

Comment:
The function name is incorrect, it does not enable font smoothing but instead sets the font smoothing value based on the parameter.

Reply:
Correct, that's a horrible name. I'll fix that.